spk/debian-chroot/src/sources.list

```diff
@@ -1,31 +1,33 @@
-################################################
-## wheezy
-deb http://ftp.fr.debian.org/debian/ wheezy main contrib non-free
-deb-src http://ftp.fr.debian.org/debian/ wheezy main contrib non-free
-
-## wheezy security
-deb http://security.debian.org/ wheezy/updates main contrib non-free
-deb-src http://security.debian.org/ wheezy/updates main contrib non-free
-
-# wheezy update
-deb http://ftp.fr.debian.org/debian/ wheezy-updates main contrib non-free
-deb-src http://ftp.fr.debian.org/debian/ wheezy-updates main contrib non-free
+# Whatever's uncommented first in this file counts as the "main" distribution
+# (unless you change the ordering). See /etc/apt/preferences for more
+# information on how pinning is set up.
 
 ################################################
 ## jessie
-deb http://ftp.fr.debian.org/debian/ jessie main contrib non-free
-deb-src http://ftp.fr.debian.org/debian/ jessie main contrib non-free
-
-## jessie security
+deb http://ftp.debian.org/debian/ jessie main contrib non-free
```

Comment:
It's bad netiquette to use the central servers. Use mirrors where possible, which is why `fr` was used.
Advanced users should probably change the defaults at some point anyway (unless they live in France...)